README.md

```diff
@@ -2141,6 +2142,92 @@ def reply_raw() -> manual[RawReply]:
 
 [Not yet implemented.](https://github.com/demergent-labs/azle/issues/496)
 
+### Timer APIs
+
+-   [clear timer](#clear-timer)
+-   [set timer](#set-timer)
+-   [set timer interval](#set-timer-interval)
+
+#### clear timer
+
+Examples:
+
+-   [timers](/examples/timers)
+
+```python
+from kybra import ic, TimerId, update
+
+@update
+def clear_timer(timer_id: TimerId):
+    ic.clear_timer(timer_id)
+```
+
+#### set timer
+
+Examples:
+
+-   [timers](/examples/timers)
+
+```python
+from kybra import Duration, ic, TimerId, update
+
+
+@update
+def set_timers(delay: Duration) -> list[TimerId]:
```

Comment:
To match the other example, this should not be a list of TimerId, this should be a tupel with two timerIds:

```python
-> tuple[TimerId, TimerId]
```